app/api/getChannelCasts/route.ts

```diff
@@ -0,0 +1,22 @@
+import getChannelsCasts from '@/lib/neynar/getChannelCasts';
+import { NextRequest, NextResponse } from 'next/server';
+
+
+export async function GET(req: NextRequest) {
+  const id = req.nextUrl.searchParams.get('channelId');
+  const startDate = req.nextUrl.searchParams.get('startDate');
```

Comment:
can we remove `startDate` param to read from supabase in the same way we already do in `getNewCasts` job?